### DISCUSSION TITLE
LangVersion = latest ?

### DISCUSSION THREAD

**Discussion by TonyValenti** [oqtane/oqtane.framework#832] [Ideas] [answered]
Would a PR updating the LangVersion to ```latest``` be accepted?

**Comment by sbwalker** [comment]
In the latest Blazor templates in .NET 5 Microsoft has removed the LangVersion attribute - so perhaps we should remove it as well?

**Comment by TonyValenti** [comment]
I think keeping it is a good idea because it makes it easy to switch between langversion = latest and langversion = preview for testing, however, I am fine with that approach because my goal is to get off of 7.3.

Which would you like?

**Comment by sbwalker** [comment]
I would say remove it

**Comment by hishamco** [comment, marked as answer]
The PR #834 is created based on the Shaun's first comment
